### PR TITLE
[DO NOT REVIEW] Issue/435 Switch country flags to use emoji character

### PR DESCRIPTION
**Pull request by elibud**
**Fixes:** #435 

**To test:** 
Run the demo and scroll down to the Countries section. 
Tap on View All to verify the full countries view.

simulator screen shot mar 15 2017 15 16 53

### CODE REVIEW

**Comment by koke** [comment]
Including the emoji in the country name, doesn't seem to play well with RTL languages:

simulator screen shot 16 mar 2017 10 10 35

File: WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m [hunk 1]
```diff
@@ -1067,16 +1067,12 @@ - (NSURLSessionDataTask *)operationForCountryForDate:(NSDate *)date
                 countryName = [countryInfoDict[key] stringForKey:@"country_full"];
             }
             StatsItem *statsItem = [StatsItem new];
-            statsItem.label = countryName;
+            statsItem.label = [NSString stringWithFormat:@"%@ %@",
```
Comment (astralbodies) [inline WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m]: I think the RTL problem can be fixed by using `[NSString localizedStringWithFormat:]`

Comment (koke) [inline WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m]: I believe that just applies the localized date/number formatters to arguments, but won't do anything about text direction. Bidirectional Text is complicated and painful, but the gist of it is that UIKit will decide on the label direction by looking at the first characters of the string.

**Comment by elibud** [comment]
Will be addressed here: https://github.com/wordpress-mobile/WordPressCom-Stats-iOS/pull/440

**Pull request closed without merging**